platform/viewer/cypress/integration/common/OHIFStudyList.spec.js

```diff
@@ -0,0 +1,328 @@
+    // });
+    // });
+
+    // //TO-TO: This test should be uncommented once issue #1114 is fixed: https://github.com/OHIF/Viewers/issues/1114
+    // it('searches Description with lower case', function() {
+    //   cy.get('@studyDescription').type('ct chest');
+    // //Wait result list to be displayed
+    // cy.waitStudyList();
+    // cy.get('@searchResult').should($list => {
+    //   expect($list.length).to.be.eq(7);
+    // });
+    // });
+
+    // //TO-TO: This test should be uncommented once issue #1114 is fixed: https://github.com/OHIF/Viewers/issues/1114
+    // it('searches Description with upper case', function() {
+    //   cy.get('@studyDescription').type('CT CHEST');
+    //   //Wait result list to be displayed
+    //   cy.waitStudyList();
+    //   cy.get('@searchResult').should($list => {
+    //     expect($list.length).to.be.eq(7);
+    //   });
+    // });
+
+    it('searches Description with exact string', function() {
+      cy.get('@studyDescription').type('CHEST');
+      //Wait result list to be displayed
+      cy.waitStudyList();
+      cy.get('@searchResult').should($list => {
+        expect($list.length).to.be.eq(2);
+      });
+    });
+
+    it('changes rows per page and checks the study count', function() {
+      //Show rows per page options
+      const pageRows = [25, 50, 100];
+
+      //Check all options of rows
+      pageRows.forEach(numRows => {
+        cy.get('select').select(numRows.toString()); //Select rows per page option
+        //Wait result list to be displayed
+        cy.waitStudyList().then(() => {
+          //Compare the search result with the Study Count on the table header
+          cy.get('@studyCount')
+            .should($studyCount => {
+              expect(parseInt($studyCount.text())).to.be.at.most(numRows); //less than or equals to
+            })
+            .then($studyCount => {
+              //Compare to the number of rows in the search result
+              cy.get('@searchResult').then($searchResult => {
+                let countResults = $searchResult.length;
+                expect($studyCount.text()).to.be.eq(countResults.toString());
+              });
+            });
+        });
+      });
+    });
+
+    //TO-TO: This test should be uncommented once issue #1120 is fixed:
+    //https://github.com/OHIF/Viewers/issues/1120
+    // it('filter study list by Study Date', function() {
+    //   //Type Start and End dates
+    //   cy.get('@studyListStartDate').type('01/01/2000');
+    //   cy.get('@studyListEndDate').type('01/01/2019');
+    //   //Display all results into one page
+    //   cy.get('select').select('100');
+    //   //Checks if all expected results are displayed
+    //   //Wait result list to be displayed
+    //   cy.waitStudyList();
+    //   cy.get('@searchResult').should($list => {
+    //     expect($list.length).to.be.eq(42);
+    //   });
+    // });
+  });
+
+  context('Tablet resolution', function() {
+    beforeEach(function() {
```

Comment:
If any of these tests are identical, but with different selectors, we may want to look into a better pattern. Not a blocker for this PR; just something to think about.

Reply:
I believe that leaving it separated will make it easier to identify possible failures. :)